src/test/java/it/com/atlassian/bitbucket/jenkins/internal/config/BitbucketPluginConfigurationIT.java

```diff
@@ -155,4 +167,19 @@ private void assertCredentialsAreTracked(Credentials credentials, String expecte
                 (ItemCredentialsFingerprintFacet) fingerprint.getFacets().iterator().next();
         assertThat(facet.getItemFullName(), is(equalTo(expectedItemName)));
     }
+
+    private boolean waitFor(Supplier<Boolean> condition, long timeoutMs) {
```

Comment:
We already have a `waitFor` in our `AsyncTestUtils` so I think we should probably use that one instead of re-inventing the wheel?